### DISCUSSION TITLE
Welcome to XDC-Community.github.io Discussions!

### DISCUSSION THREAD

**Discussion by WalterBlueu** [XDC-Community/XDC-Community.github.io#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by inv4fee2020** [comment]
Hey WB.. 👋

**Reply by inv4fee2020** [reply]
I'm not a developer in the true sense but can help with adding/reviewing content. Are there any areas in particular that need attention?

**Reply by WalterBlueu** [reply]
Hey inv4fee202.  At the moment everything needs attention, I announced xdc.community a little ahead of schedule in order to get the XIP process out in the open.  I am also looking for people to add information for L2s, so if you wanted to bring documentation here for PLI node setup I think it would be an awesome fit.
